.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
```

Comment:
@firante , your PR contains too many irrelevant commits. Please create new branch out of dev and cherry-pick your commits there. Also please don't change config.js if it's not required.